api4/plugin.go

```diff
@@ -115,6 +117,8 @@ func installPluginFromUrl(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	client := c.App.HTTPService.MakeClient(true)
```

Comment:
1/5 that it would be more clean to add a `timeout time.Time` parameter to `MakeClient`.

Reply:
Yes, that was my initial thought too. However, `MakeClient` is used in several places already that currently are not interested in changing the default value so I did not want to have to update all and force to provide a value. 
The other option I was debating on was to add a `MakeClientWithTimeout` but was on the fence with adding this other method when it returned the client and it could be updated directly before using it. 

0/5 on this one, I can change it if we think it could be of any value to others in the future.

Reply:
You're right, forcing a value isn't a good idea.

We can just go with what you proposed.